games/arma3/entrypoint.sh

```diff
@@ -392,10 +392,16 @@ if [[ ${HC_NUM} > 0 ]]; then
     done
 fi
 
+# Define the log file path with a timestamp
+LOG_FILE="/home/container/logs/arma3server_$(date '+%m_%d_%Y_%H%M%S').log"
+
+# Ensure the logs directory exists
+mkdir -p /home/container/logs
```

Comment:
Should we store them with the other logs in `serverprofile/` instead, or do you think having them in a separate folder is better? Your call, but if you agree, do you mind moving this command above line 349? I just realized I have an edge case bug where the "HC cache clearing" may fail on first boot due to `serverprofile/` not existing yet.

Reply:
Yep! I'll get that changed.